CI/Dockerfile

```diff
@@ -1,9 +1,43 @@
+# Svalinn DAGMC Dockerfile
+
+# build from repository root directory with command
+# docker build -t dagmc -f CI/Dockerfile .
+
+# Global ARGS set before the first build stage are accessable by all build stages
+ARG build_mw_reg_tests=OFF
```

Comment:
I think we ignore this right now since MW Regression testing is broken

Reply:
Yes and this is tracked in an issue
https://github.com/svalinn/DAGMC/issues/858

Resolved as I have removed this build ARG